src/components/MessageBot/MessageBot.tsx

```diff
@@ -25,18 +28,31 @@ const Message: StyledComponent<{}, {}, TockTheme> = styled.div`
   ${props => (props.theme && props.theme.styles && props.theme.styles.messageBot) || ''}
 `;
 
-const MessageBot: ({ children }: { children: ReactNode }) => JSX.Element = ({
-  children,
-}: {
-  children: ReactNode;
-}) => {
-  return (
-    <MessageContainer>
-      <Message>
-        <span dangerouslySetInnerHTML={{ __html: children ? children.toString() : '' }}></span>
-      </Message>
-    </MessageContainer>
-  );
-};
+export interface MessageProps {
+  message: messageType;
+  sendAction: (button: Button) => void;
+}
+
+const MessageBot: (props: MessageProps) => JSX.Element = ({
+  message,
+  sendAction,
+}: MessageProps) => (
+  <MessageContainer>
+    <Message>
+      <span
+        dangerouslySetInnerHTML={{ __html: message.message.toString() }}
+      ></span>
+    </Message>
+    {Array.isArray(message.buttons) && message.buttons.length > 0 ? (
+      <QuickReplyList>
```

Comment:
QuickReplyList component can be renamed to ButtonList ... (can be done in another design-related task)

Reply:
We will see that in the design-related task